packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -3,16 +3,21 @@ import { ComponentPropsCompat, ComponentStateCompat, ShorthandPropsCompat } from
 import { PresenceBadgeProps, PresenceBadgeStatus } from '@fluentui/react-badge';
 
 export interface AvatarProps extends ComponentPropsCompat, React.HTMLAttributes<HTMLElement> {
-  /** The Avatar's image. */
+  /**
+   * The Avatar's image.
+   */
   image?: ShorthandPropsCompat<React.ImgHTMLAttributes<HTMLImageElement>>;
 
   /**
-   * The label shown when there's no image or icon. Defaults to the initials derived from `name` using
-   * `getInitials`.
+   * The label shown when there's no image. Defaults to the initials derived from `name` using `getInitials`.
    */
   label?: ShorthandPropsCompat<React.HTMLAttributes<HTMLElement>>;
 
-  /** Icon displayed when there's no image. */
+  /**
+   * Icon to be displayed when the avatar doesn't have an image or name (or if getInitials returns an empty string).
+   *
+   * @defaultvalue Person20Regular (the default icon's size depends on the Avatar's size)
```

Comment:
typo ? 

```suggestion
   * @defaultValue `Person20Regular` (the default icon's size depends on the Avatar's size)
```

1.  we are using standard `@default` signature in vNext ( which is a standard JSDoc pragma - also storybook understands that and generates controls with default value accordingly )
... whilst `@defaultValue` is proprietary to TSDoc -> ApiExtractor

2. token references should be escaped via backticks
3. would be nice to provide additional link ref so consumer can check those icons -> `See {@link Person20Regular} for more details.` etc

Reply:
> 1. we are using standard `@default` signature in vNext ( which is a standard JSDoc pragma - also storybook understands that and generates controls with default value accordingly )
>    ... whilst `@defaultValue` is proprietary to TSDoc -> ApiExtractor

If we want to enforce this we should probably add a lint rule for it. It's unfortunate for interoperability purposes that API Extractor chose a different tag ([rationale here](https://github.com/microsoft/tsdoc/issues/27#issuecomment-423416355) which kind of makes sense, but didn't seem to consider tooling interoperability).

Reply:
Logged #19259 to update all the comments in Avatar to replace `@defaultvalue`